drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -554,5 +556,17 @@ def ce_referral_step(id:)
 
       Hmis::WorkflowExecution::Step.viewable_by(current_user).find(id)
     end
+
+    # Globally available CE opportunities gated by admin permission
+    ce_opportunities_field
+    def ce_opportunities(**args)
+      # TODO(#7506) - gated by admin permission
+      resolve_ce_opportunities(Hmis::Ce::Opportunity.all, **args)
```

Comment:
Oh didn't think to call this out in the spec, but this needs to be limited to Opportunities that belong to Projects in the current Data Source. Per https://github.com/open-path/Green-River/issues/6612 we never want to be resolving data from other data sources

Reply:
Sorry I lost track of this @martha, your implementation looks good to me.